common/meta/raft-store/src/config.rs

```diff
@@ -159,12 +153,6 @@ impl RaftConfig {
     }
 
     pub fn check(&self) -> common_exception::Result<()> {
-        if self.boot && self.single {
-            return Err(ErrorCode::InvalidConfig(
-                "--boot and --single can not be both set",
-            ));
-        }
-
         Ok(())
     }
```

Comment:
lgtm, but in this case, I think we can remove the `check`

Reply:
Hmm... Yes it's a little bit dirty in this commit.
So let me fill in some useful stuff later. Such as `--single` and `--join` can not be specified together, or `--join` must not be set to itself.

Reply:
ok